packages/graph/hash_graph/lib/graph/src/api/web/api_resource.rs

```diff
@@ -0,0 +1,10 @@
+use axum::Router;
+
+use crate::datastore::Datastore;
+
+pub(super) trait RoutedResource: utoipa::OpenApi {
```

Comment:
Does this need a doc-comment?

Reply:
Yes for sure, addressed in https://github.com/hashintel/hash/pull/750/commits/1e38dc07256d56933677192a53d2af97dc18b20a